apps/docs/libraries/ts/ratelimit/override/delete-override.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: "Delete Override"
+description: "Deletes an override"
+---
+
+## Request
+<ParamField body="identifier" type="string" required>
+Identifier of your user, this can be their userId, an email, an ip or anything else. Wildcards ( * ) can be used to match multiple identifiers, More info can be found at https://www.unkey.com/docs/ratelimiting/overrides#wildcard-rules
+</ParamField>
+
+<Warning>
+Either `namespaceId` or `namespaceName` is required. Not both.
+</Warning>
+
+<ParamField body="namespaceId" type="string" required>
+The id of the namespace. Either namespaceId or namespaceName must be provided
+</ParamField>
+
+<ParamField body="namespaceName" type="string" required>
+Namespaces group different limits together for better analytics. You might have a namespace for your public API and one for internal tRPC routes. Wildcards can also be used, more info can be found at https://www.unkey.com/docs/ratelimiting/overrides#wildcard-rules
+</ParamField>
+
+
+## Response
+
+No response, but if no error is returned the override has been deleted successfully.
+
+
+<RequestExample>
+
+```ts
+await unkey.deleteOverride({
```

Comment:
A request should show accurately how it works. 

Do you need all three params to delete it?

Reply:
Sure dont, Just one of namespace options